presentation/common/src/presentation-common/content/Descriptor.ts

```diff
@@ -55,6 +65,26 @@ export namespace SelectClassInfo {
       relatedInstanceClasses: json.relatedInstanceClasses.map((p) => RelatedClassInfo.fromJSON(p)),
     };
   }
+
+  export function decompressSelectClassInfoJSON(compressedSelectClass: CompressedSelectClassInfoJSON, classesMap: { [id: string]: CompressedClassInfoJSON }): SelectClassInfoJSON {
+    return {
+      ...compressedSelectClass,
+      selectClassInfo: {id: compressedSelectClass.selectClassInfo, ...classesMap[compressedSelectClass.selectClassInfo]},
+      navigationPropertyClasses: compressedSelectClass.navigationPropertyClasses.map((compressedInfoJSON) => decompressRelatedClassInfoJSON(compressedInfoJSON, classesMap)),
+      relatedInstanceClasses: compressedSelectClass.relatedInstanceClasses.map((compressedInfoJSON) => decompressRelatedClassInfoJSON(compressedInfoJSON, classesMap)),
+      pathToPrimaryClass: compressedSelectClass.pathToPrimaryClass.map((compressedInfoJSON) => decompressRelatedClassInfoJSON(compressedInfoJSON, classesMap)),
+      relatedPropertyPaths: compressedSelectClass.relatedPropertyPaths.map((path) => path.map((compressedInfoJSON) => decompressRelatedClassInfoJSON(compressedInfoJSON, classesMap))),
+    };
+  }
+
+  export function decompressRelatedClassInfoJSON(compressedInfoJSON: CompressedRelatedClassInfoJSON, classesMap: { [id: string]: CompressedClassInfoJSON }): RelatedClassInfoJSON {
```

Comment:
Why is this under `SelectClassInfo` namespace?

Reply:
Moved it to be under RelatedClassInfo namespace